stable/kong/templates/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: "{{ template "kong.fullname" . }}"
+  labels:
+    app: "{{ template "kong.name" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    component: "kong_server"
```

Comment:
It doesn't look like you have real components in your chart. There is only a single deployment. Usually, components are used to identify multiple deployments in a chart and then are added to label selectors. This is not the case here. Do they server any special purpose?